app/code/Magento/Cms/Ui/Component/Form/Field/StoreView.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Cms\Ui\Component\Form\Field;
+
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Framework\View\Element\UiComponentFactory;
+use Magento\Framework\View\Element\UiComponentInterface;
+use Magento\Store\Model\StoreManagerInterface as StoreManager;
+use Magento\Ui\Component\Form\Field;
+
+/**
+ * Check to disable store view field
+ *
+ * Class \Magento\Cms\Ui\Component\Form\Field\StoreView
+ */
+class StoreView extends Field
```

Comment:
I am not sure if this class should be placed right here.
Could we move this to `Magento\Store\Ui\Component\Form\Field\StoreView`?